Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -99,7 +76,7 @@ private void UpdateMouseRotation(Vector3 mouseDeltaRotation)
                 timeoutTimer = 0.0f;
             }
 
-            transform.Rotate(mouseDeltaRotation, Space.World);
+            transform.Rotate(mouseDeltaRotation, Space.Self);
```

Comment:
Have you tested in an immersive headset as well to verify some of these changes?

Reply:
good call. will do that today

Reply:
confirmed behavior as expected in vr